fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItemSettings.java

```diff
@@ -53,6 +53,18 @@ public FabricItemSettings customDamage(CustomDamageHandler handler) {
 		return this;
 	}
 
+	/**
+	 * Sets this item as a fuel, with a provided {@code cookTime}.
+	 *
+	 * @param cookTime the number of ticks before this item will be consumed in a furnace
+	 * @return this builder
+	 * @see net.fabricmc.fabric.api.registry.FuelRegistry
+	 */
+	public FabricItemSettings fuel(int cookTime) {
```

Comment:
Hmm, the content registry module calls it "fuel burn time" which might be better (it's not only used for cooking after all).

Reply:
Oops yeah `cook time` is for furnace inputs, not furnace fuels.

Reply:
The module calls it "cook time" actually https://github.com/FabricMC/fabric/blob/f9b3753bc0fc312e5dd20036df77d4d31f01480b/fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/impl/content/registry/FuelRegistryImpl.java#L67

Reply:
It should probably be changed in the module then imo, yarn calls it `burnTime` if you look into `AbstractFurnaceBlockEntity`. Out of scope for this PR obviously, but feel free to open another one.

Reply:
https://github.com/FabricMC/fabric/blob/f9b3753bc0fc312e5dd20036df77d4d31f01480b/fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/FuelRegistry.java#L23

The *API* of that module calls it "fuel burn time".

Reply:
Renamed it